src/ngraph/runtime/gpu/pass/gpu_rnn_fusion.cpp

```diff
@@ -313,9 +313,6 @@ void ngraph::runtime::gpu::pass::LSTMFusion::construct_lstm_fprop()
         auto ht_output = std::make_shared<op::GetOutputElement>(lstm, 0);
         auto ct_output = std::make_shared<op::GetOutputElement>(lstm, 2);
 
-        NGRAPH_CHECK(lstm->get_outputs().at(0).get_inputs().size() == 2,
-                     "Lstm node doesnt have two outputs");
-
         // Now identify the nodes which consumes the output of LSTM nodes
         // and replace them accordingly
         // find the user's for {ht|ct} and replace them with lstm_goe_1
```

Comment:
Actually might want to reword the whole comment; "find the user's" makes this read like it needs an object.

Reply:
The comment is being deleted.